config/docker/fragment/kselftest.jinja2

```diff
@@ -1,10 +1,16 @@
 # kselftest
 {%- if sub_arch %}
-{%- set pkgarch = ':' + sub_arch %}
-RUN dpkg --add-architecture {{ sub_arch }}
+    {%- if sub_arch == 'riscv64' %}
+        {%- set libc6_pkgarch = '-' + sub_arch + '-cross' %}
+        {%- set pkgarch = '' %}
+    {% else %}
+        {%- set libc6_pkgarch = ':' + sub_arch %}
+        {%- set pkgarch = ':' + sub_arch %}
```

Comment:
nit: this would be a little clearer to read if you just `set pkgarch = libc6_pkgarch`

Reply:
changed